forms/uds/a1a/form_a1a_ivp_error_checks_p.csv

```diff
@@ -0,0 +1,12 @@
+error_code,error_no,error_type,form_name,packet,var_name,check_type,test_name,short_desc,full_desc,test_logic,comp_forms,comp_vars,do_in_redcap,in_prev_versions,questions
+a1a-ivp-p-1001,1001,Alert,a1a,I,TRANSPROB,Plausibility,Consistency in transportation access across forms,If TRANSPROB or TRANSWORRY = 1 (often) and Form A1 TRANSPOR = 4 (never limits me),If Q3. TRANSPROB (Transportation problems in past month?) or Q4. TRANSWORRY (Transportation worries in past month?) = 1 (often) and Form A1 Q23. TRANSPOR (lack of transportation) = 4 (never limits me),If (TRANSPROB=1 or TRANSWORRY=1) and TRANSPOR=4,a1,"TRANSPROB, TRANSWORRY, TRANSPOR",No,No,
+a1a-ivp-p-1002,1002,Alert,a1a,I,TRANSPROB,Plausibility,Consistency in transportation access across forms,If TRANSPROB or TRANSWORRY = 3 (never) and Form A1 TRANSPOR is 1 (always limits me),If Q3. TRANSPROB (Transportation problems in past month?) or Q4. TRANSWORRY (Transportation worries in past month?) = 3 (often) and Form A1 Q23. TRANSPOR (lack of transportation) = 1 (always limits me),If (TRANSPROB=3 or TRANSWORRY=3) and TRANSPOR=1,a1,"TRANSPROB, TRANSWORRY, TRANSPOR",No,No,
+a1a-ivp-p-1003,1003,Alert,a1a,I,BILLPAY,Plausibility,Consistency in income across forms,"IF BILLPAY = 4 or 5 (very or extremely), and Form A1 INCOME 3 or 4 (have excess income)","IF Q7. BILLPAY (Difficulty with monthly payments?) = 4 or 5 (very or extremely), and Form A1 Q18. INCOME (household income) = 3 or 4 (have excess income)","If BILLPAY in (4,5) and INCOME in (3,4)",a1,"BILLPAY, INCOME",No,No,
+a1a-ivp-p-1004,1004,Alert,a1a,I,BILLPAY,Plausibility,Consistency in income across forms,"IF BILLPAY = 1 (not at all), and Form A1 INCOME = 1 (insufficient to meet needs)","IF Q7. BILLPAY (Difficulty with monthly payments?) = 1 (not at all), and Form A1 Q18. INCOME (household income) = 1 (insufficient to meet needs)",If BILLPAY=1  and INCOME=1,a1,"BILLPAY, INCOME",No,No,

```

Comment:
Same as above, Form A1 doesn't have a variable named INCOME? A1. 18 is a different question.